### PR TITLE
Implement length argument in lorem.word

### LINKED ISSUE #509: lorem.word(num) always return random length word
image
The length of word should equal the input number, but actually Not.

### PR DESCRIPTION
**Pull request by jreina**
Adds unit tests for lorem.word both with and without supplying the length parameter.
Implements the length parameter of lorem.word.
Resolves #509

### CODE REVIEW

**Comment by hbrysiewicz** [comment]
+1

**Comment by rmoskal** [comment]
Hello?

**Comment by MatissJanis** [comment]
merge please

File: lib/lorem.js [hunk 1]
```diff
@@ -8,13 +8,17 @@ var Lorem = function (faker) {
...
    * @method faker.lorem.word
-   * @param {number} num
+   * @param {number} length length of the word that should be returned. Defaults to a random length
    */
-  self.word = function (num) {
-    return faker.random.arrayElement(faker.definitions.lorem.words);
+  self.word = function (length) {
+    var hasRightLength = function(word) { return word.length === length; };
+    var properLengthWords = typeof length == 'undefined'
+    ? faker.definitions.lorem.words
```
Comment (Marak) [inline lib/lorem.js:19]: We try not to use conditional assignment for `faker.js`. 

Could you update this to not use to the conditional assignment operator?

**Pull request merged**